Gems/PhysX/Code/Source/EditorShapeColliderComponent.cpp

```diff
@@ -64,6 +65,17 @@ namespace PhysX
         return AZ::Edit::PropertyVisibility::Hide;
     }
 
+    AZ::Crc32 EditorShapeColliderComponent::SingleSidedVisibility()
```

Comment:
suggestion: It looks like this could be `const`?

Reply:
Added, and added to the other existing visibility function as well.